docs/releases/4.0.txt

```diff
@@ -229,7 +229,11 @@ Models
 Requests and Responses
 ~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* The :class:`~django.middleware.security.SecurityMiddleware` now adds the
+  :ref:`Cross-Origin Opener Policy <cross-origin-opener-policy>` header with a
+  value of ``'same-origin'`` to prevent cross-origin popups from sharing the
+  same browsing context. You can prevent this header from being added by
+  setting the :setting:`SECURE_CROSS_ORIGIN_OPENER_POLICY` setting to ``None``.
```

Comment:
I moved this to the "Security" section.

Reply:
I think it's better to list security enhancements in their respective components (contrib.auth, etc.) rather than grouping them separately.

Reply:
In recent releases, we've included such changes in the "Security" section :thinking: (see [3.1](https://docs.djangoproject.com/en/dev/releases/3.1/#security), [3.0](https://docs.djangoproject.com/en/dev/releases/3.0/#security)).

Reply:
I too think it makes sense in "Security"

Reply:
Wouldn't changes like PBKDF2 iteration increases be considered "security"? or "The admin now installs a final catch-all view that redirects unauthenticated users to the login page, regardless of whether the URL is otherwise valid. This protects against a potential model enumeration privacy issue."? (these were listed under contrib.auth and contrib.admin respectively).

I think if a change doesn't fit into some other category it could be labeled "security." Otherwise, I wouldn't group all security changes separately.

Reply:
Fair enough. I think "requests and responses" is fine too.

Reply:
OK